docker/server/README.md

```diff
@@ -33,6 +33,22 @@ ClickHouse configuration represented with a file "config.xml" ([documentation](h
 $ docker run -d --name some-clickhouse-server --ulimit nofile=262144:262144 -v /path/to/your/config.xml:/etc/clickhouse-server/config.xml yandex/clickhouse-server
 ```
 
+## How to extend this image
+
+If you would like to do additional initialization in an image derived from this one, add one or more `*.sql`, `*.sql.gz`, or `*.sh` scripts under `/docker-entrypoint-initdb.d`. After the entrypoint calls `initdb` it will run any `*.sql` files, run any executable `*.sh` scripts, and source any non-executable `*.sh` scripts found in that directory to do further initialization before starting the service.
+
+For example, to add an additional user and database, add the following to `/docker-entrypoint-initdb.d/init-db.sh`:
+
+```bash
+#!/bin/bash
+set -e
+
+clickhouse client -n <<-EOSQL
```

Comment:
Better use clickhouse-client

Reply:
There is no `clickhouse-client` binary inside the `clickhouse-server` image.

Reply:
And that is very confusing. IMHO client should be added to server container. It costs nothing (just a symlink) but makes usage of clickhouse-client much more logical.